src/Coordination/KeeperStateManager.cpp

```diff
@@ -47,18 +33,19 @@ bool isLocalhost(const std::string & hostname)
 
 std::unordered_map<UInt64, std::string> getClientPorts(const Poco::Util::AbstractConfiguration & config)
 {
-    static const char * config_port_names[] = {
-        "keeper_server.tcp_port",
-        "keeper_server.tcp_port_secure",
-        "interserver_http_port",
-        "interserver_https_port",
-        "tcp_port",
-        "tcp_with_proxy_port",
-        "tcp_port_secure",
-        "mysql_port",
-        "postgresql_port",
-        "grpc_port",
-        "prometheus.port",
+    using namespace std::string_literals;
+    static const std::array config_port_names = {
```

Comment:
each time I read your PR I learn something new about C++.

Reply:
Glad to hear that 😄